apiary.apib

```diff
@@ -2618,6 +2618,23 @@ Get a list of products.
                         + description: `dark chocolate` (string)
                 + external_id: `123` (string)
 
+### products.add [POST /products.add]
+
+ Add a new product.
+
+ + Request (application/json;charset=utf-8)
+
+     + Attributes (object)
+        + name: `cookies` (string, optional)
```

Comment:
We need to specify the code and name should be OneOf

Reply:
You need to provide at least a name or a code, but you can provide both as well. No idea how you can document this in apiary though. @jenssegers , any idea?

Reply:
Is that a rule we can work around and make the name required and code optional?

Reply:
Anything happening with this?

Reply:
We have some `OneOf` things in the documentation somewhere, just can't remember where. But it is for sure possible.

Reply:
Invoices.draft 
+ for_attention_of (object, optional)
                + One Of
                    + name: `Finance Dept.` (string, required)
                    + contact_id: `417a2231-c3c7-4e1c-a6bb-1b014836ca60` (string, required)